lua/pipe.lua

```diff
@@ -29,18 +29,170 @@ ffi.cdef [[
 	
 	// DPDK SPSC ring
 	struct rte_ring { };
+	//struct rte_mbuf { };
```

Comment:
this seems odd; did we define this twice somewhere accidentally?

Reply:
Not sure where this came from.  Removing the commented line.